lib/m2ts/caption-stream.js

```diff
@@ -838,16 +878,24 @@ Cea608Stream.prototype.shiftRowsUp_ = function() {
   for (i = 0; i < this.topRow_; i++) {
     this.displayed_[i] = '';
   }
+  for (i = this.row_ + 1; i < BOTTOM_ROW + 1; i++) {
+    this.displayed_[i] = '';
+  }
   // shift displayed rows up
-  for (i = this.topRow_; i < BOTTOM_ROW; i++) {
+  for (i = this.topRow_; i < this.row_; i++) {
     this.displayed_[i] = this.displayed_[i + 1];
   }
   // clear out the bottom row
-  this.displayed_[BOTTOM_ROW] = '';
+  this.displayed_[this.row_] = '';
 };
 
-// paintOn mode is not implemented
-Cea608Stream.prototype.paintOn = function() {};
+Cea608Stream.prototype.paintOn = function(pts, text) {
+  var baseRow = this.displayed_[this.row_];
+
+  baseRow = baseRow.slice(0, this.column_) + text +
```

Comment:
Using `this.column_` here will probably break mid-row formatting, so I'm going to mark this PR as WIP until I decide what I want to do here.

Reply:
I've decided to defer this functionality until I implement proper caption positioning. Removing WIP, this is ready to go.